src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,107 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
-    /**
-     * Return the current date as a String depending on a query.
-     *
-     * The query can be passed for the whole date or for it's part:
-     *      - FULL - current date as a whole: year, month, day of month
-     *        formatted as `YYYY-MM-DD` (a default return value);
-     *      - YEAR - current year;
-     *      - MONTH - name of the current month;
-     *      - DAY - current day of month;
-     * In any other case throw DateTimeException.
-     **/
+    private static final String CUSTOM_FORMAT = "d MMM yyyy";
+    private static final String FORMAT_DATE_AND_TIME = "dd MMMM yyyy H:mm";
+    private static final String FORMAT_PARSE_DATE = "yyyyMMdd";
+
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+
+        switch (datePart) {
+            case DAY: return String.valueOf(LocalDate.now().getDayOfMonth());
```

Comment:
Avoid calling LocalDate.now multiple times in one method. Let’s create a variable in the method for this purpose.
https://mate-academy.github.io/jv-program-common-mistakes/java-core/java-eight-part-two/date-time-api